build.sbt

```diff
@@ -129,6 +134,7 @@ lazy val defaultScalacOptions = scalacOptions ++= Seq(
 lazy val defaultLibraryDependencies = libraryDependencies ++= Seq(
   scalaOrganization.value % "scala-reflect" % scalaVersion.value % Provided,
   scalaOrganization.value % "scala-compiler" % scalaVersion.value % Provided,
-  "org.scalacheck" %%% "scalacheck" % "1.14.0" % Test,
-  "org.scalatest" %%% "scalatest" % "3.0.8" % Test
+  "org.scalacheck" %%% "scalacheck" % "1.14.3" % Test,
+  "org.scalatest" %%% "scalatest" % "3.1.1" % Test,
+  "org.scalatestplus" %%% "scalacheck-1-14" % "3.1.1.0" % Test,
```

Comment:
The current version is `3.1.1.1`

Reply:
Just push an update for this (and Scala.js to 1.0.1), thanks!